### PR TITLE
Button: add `aria-disabled` support

### PR DESCRIPTION
**Pull request by jakobe**
## Which issue does this PR close?

This PR closes no issue, but is a pre-burner to #3681 

## What is the new behavior?

Add support for `aria-disabled` on `Button` component:

Setting `aria-disabled="true"` on the `Button` component will render the button as disabled and prevent activation through mouse, touch and keyboard but still allow the button to be focused and therefore percieveable.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Reminders
- [x] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] Request that the changes are code-reviewed 
- [x] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: apps/cookbook/src/app/showcase/button-showcase/button-showcase.component.html [hunk 1]
```diff
@@ -142,6 +142,47 @@ <h2>Disabled</h2>
...
+  <p>
+    The
+    <code>disabled</code>
+    attribute effectively hides the button from assistive technologies, such as screen readers, by
+    removing the button from the focus order of the web page.
+  </p>
+  <p>
+    In many scenarios it's good practice to expose the button as disabled, but still available for
+    users to find when navigating via the
+    <kbd>Tab</kbd>
```
Comment (RasmusKjeldgaard) [inline apps/cookbook/src/app/showcase/button-showcase/button-showcase.component.html:155]: Nice `kbd` 🥇

File: libs/designsystem/button/src/button.component.spec.ts [hunk 2]
```diff
@@ -87,6 +87,46 @@ describe('ButtonComponent', () => {
...
+        expect(element).toHaveComputedStyle({
+          'background-color': getColor('semi-light'),
+        });
+      });
+
+      it('should render with correct color', () => {
+        expect(element).toHaveComputedStyle({
+          color: getColor('semi-dark', 'shade'),
+        });
+      });
```
Comment (RasmusKjeldgaard) [inline libs/designsystem/button/src/button.component.spec.ts:105]: These are a little redundant with the visual tests, but I can see how it follows our approach everywhere else. 

So not important, but consider removing these to not keep piling `toHaveComputedStyle` tests on top.

File: apps/cookbook/src/app/showcase/button-showcase/button-showcase.component.html [hunk 3]
```diff
@@ -142,6 +142,47 @@ <h2>Disabled</h2>
...
+  <h3>Accessible Disabled Buttons</h3>
+  <p>
+    The
+    <code>disabled</code>
+    attribute effectively hides the button from assistive technologies, such as screen readers, by
+    removing the button from the focus order of the web page.
+  </p>
+  <p>
+    In many scenarios it's good practice to expose the button as disabled, but still available for
+    users to find when navigating via the
```
Comment (RasmusKjeldgaard) [inline apps/cookbook/src/app/showcase/button-showcase/button-showcase.component.html]:
```suggestion
    In many scenarios it's good practice to expose the button as disabled, but still make it available for
    users to find when navigating via the
```

File: apps/cookbook/src/app/showcase/button-showcase/button-showcase.component.html [hunk 4]
```diff
@@ -142,6 +142,47 @@ <h2>Disabled</h2>
...
+    <code>disabled</code>
+    will render the button perceivable but disabled.
+  </p>
+  <p>
+    <a
+      class="kirby-external-icon"
+      target="_blank"
+      href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled"
+    >
+      Read more on aria-disabled on MDN
```
Comment (RasmusKjeldgaard) [inline apps/cookbook/src/app/showcase/button-showcase/button-showcase.component.html]: The double "on" is a bit off for me but completely up to you!
```suggestion
      Read more about aria-disabled on MDN
```

**Review by RasmusKjeldgaard — COMMENTED** [review]
Amazing addition!

File: libs/designsystem/button/src/button.component.spec.ts (hunk 2 above)
Comment (jakobe) [reply libs/designsystem/button/src/button.component.spec.ts:105]: Hmmm true - we should probably remove most of the tests that expect on style.
I'll leave it for now and we can do a separate clean PR later on...

**Review by RasmusKjeldgaard — APPROVED** [review]

**Pull request merged**